Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_True_Positive_Incident_Handling.yml

```diff
@@ -57,7 +55,7 @@ inputs:
   playbookInputQuery:
   required: false
   value:
-    simple: ${incident.processpath}
+    simple: ${incident.processpaths}
```

Comment:
if we can get multiple values here, lets add unique tranformer no?

Reply:
Done.